modules/support-services/configuring/src/main/resources/config/geonetwork-dev.properties

```diff
@@ -1,14 +0,0 @@
-jwt.secret=geonetwork
```

Comment:
Where are those files now ? I don't get it.

Reply:
`conf/application.yml`. That one contains common config for all services. Additional settings go on each service's `<service-name>.yml`